KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js

```diff
@@ -0,0 +1,489 @@
+define({ 
+
+devHeight :null,
+thisCard :null,
+thisCardIndex:null,
+cardFrameRel :null,
+
+frmAgendaPreshow : function(){
+  var self = this;
+  this.addActionToSessionTiles();
+  //this.view.tileOuterCopy.isVisible=false;
+  this.view.referenceAgenda.isVisible=false;
+  this.view.referenceSession.isVisible=false;
+  this.view.postShow=this.frmAgendaPostshow;
+  
+  this.view.sessionContentContainer.top="100%"
+  
+  this.view.buttonBack.isVisible=false;
+  this.view.imageBack.opacity=0;
+  
+  this.view.buttonBack.onClick=this.frmAgendaSessionClose;
+  
+  this.view.detailsScroller.onScrolling=this.detailsScrollerOnScrolling;
+  
+  this.view.filterAll.onClick=function(eventobject){
+    self.agendaFilter(eventobject);
+  }
+  this.view.filterDBX.onClick=function(eventobject){
+    self.agendaFilter(eventobject);
+  }
+  this.view.filterQuantum.onClick=function(eventobject){
+    self.agendaFilter(eventobject);
+  }
+  this.view.sessionTileAnim.anchorPoint={"x":.5,"y":0};
+  //this.view.sessionTile.tileBGImageKony.anchorPoint={"x":1,"y":.5};
```

Comment:
remove commented code

Reply:
Will discuss with Ed and will remove after the discussion